packages/core-utils/src/route.ts

```diff
@@ -220,15 +234,15 @@ function getRouteTypeComparatorValue(route) {
  * Calculates the sort comparator value given two routes based off of route type
  * (OTP mode).
  */
-function routeTypeComparator(a, b) {
+function routeTypeComparator(a: Route, b: Route): number {
   return getRouteTypeComparatorValue(a) - getRouteTypeComparatorValue(b);
 }
 
 /**
  * Determines whether a value is a string that starts with an alphabetic
  * ascii character.
  */
-function startsWithAlphabeticCharacter(val) {
+function startsWithAlphabeticCharacter(val: unknown): boolean {
```

Comment:
Add at least a FIXME. I think route is `shortName ?: string | number` but don't see an existing definition.

Reply:
I think here again an explicit `unknown` is ok. If you were to pass say a function in, this method would correctly return `false`